src/lib/UtapiClient.js

```diff
@@ -80,27 +84,30 @@ export default class UtapiClient {
     _noop() {}
 
    /**
-    * Check the types of optional `params` object properties. This enforces
-    * object properties for particular push metric calls and property types.
+    * Check the types of `params` object properties. This enforces object
+    * properties for particular push metric calls and enforces the existence of
+    * at least one metric type (e.g., 'bucket' or 'account').
     * @param {object} params - params object with metric data
-    * @param {string} params.bucket - bucket name
+    * @param {string} params.bucket - (optional) bucket name
```

Comment:
you can put in brackets around `params.bucket` to indicate it's optional
http://usejsdoc.org/tags-param.html#optional-parameters-and-default-values

same for the other optional params